### ISSUE TITLE
CSS issue with 15.5.2 

### ISSUE THREAD

**Issue by jakebern** [airbnb/react-dates#903]
Updated from 15.2.1 to 15.5.2 and had breaking changes.  I updated moment - is there another dependency I'm missing?

With following packages, no issues: (see screenshot)
```json
"dependencies": {
 "moment": "^2.19.1",
    "react": "^16.2.0",
    "react-dates": "^15.2.1",
    "react-dom": "^16.0.0",
}

With  15.5.2, getting breakages (see screenshot). 
  "dependencies": {
    "moment": "^2.20.0",
    "react": "^16.2.0",
    "react-dates": "^15.5.2",
    "react-dom": "^16.2.0",
  }
```

15.2.1
<img width="719" alt="15 2 1" src="https://user-images.githubusercontent.com/5565911/34076799-d629365a-e334-11e7-97b7-a3a437a5065c.png">

15.5.2
<img width="1013" alt="15 5 2" src="https://user-images.githubusercontent.com/5565911/34076802-ddf68b44-e334-11e7-9a34-7fecdadbe2b0.png">

**Comment by ljharb** [comment]
hmm, do you see any console errors?

It's certainly possible that something went wrong with the build process for v15.5.2. Can you check on v15.5.1 and confirm?

**Comment by majapw** [comment]
I just looked at https://unpkg.com/react-dates@15.5.2/lib/css/ and it looks
like the css file didn't build properly. @ljharb can you take a look and
republish when you get a chance? I know that there is a version of
react-with-styles that the css interface doesn't work with. Can you check
what `npm build:css` outputs?

**Comment by ljharb** [comment]
@majapw ah, that's totally it; pegging to react-with-styles 2.2.0 and republishing shortly. When are we reverting that change in 2.x? O.o

**Comment by ljharb** [comment]
v15.5.3 is released, and v15.5.2 is deprecated.
